### PR TITLE
WFLY-4818 WS-RM persistence with RMTxStore does not work with JNDI Lookup

### PR DESCRIPTION
**Pull request by TomasHofman**
Adding test case to the issue.

This is expected to fail builds, and will need to be retested after Apache CXF is upgraded to 3.0.6.

### CODE REVIEW

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ws/wsrm/RMStoreFeature.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package org.jboss.as.test.integration.ws.wsrm;
```
Comment (bstansberry) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ws/wsrm/RMStoreFeature.java]: Needs a copyright header.

**Comment by bstansberry** [comment]
@asoldano. Please approve. Thanks!

**Comment by asoldano** [comment]
@bstansberry to me this PR should stay pending till we actually move to CXF 3.0.6, which won't happen in few days. We might also merge the changes and exclude the test, but I'd rather see it pass with the CXF component upgrade before merging.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ws/wsrm/RMStoreFeature.java (hunk 1 above)
Comment (TomasHofman) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ws/wsrm/RMStoreFeature.java]: Fixed, thanks.

**Comment by bstansberry** [comment]
Unless this CXF upgrade is coming w/in a week, please close this and open a new PR when the upgrade is ready.

**Pull request closed without merging**